media-libs/libffado-2.3.0: New package

marcan opened the pull request:
FFADO is a userspace driver for IEEE1394 (FireWire) audio interfaces.

From audio-overlay: https://github.com/gentoo-audio/audio-overlay
(Previously from the proaudio overlay)

Gentoo-Bug: https://bugs.gentoo.org/622520

This is a new package and I'm happy to proxy-maintain it.

This is intended to become a dependency of media-sound/jack-audio-connection-kit. In order to avoid the currently existing automagic dependency, I'm also explicitly forcing FFADO support off in those ebuilds in this PR. I'll open a separate PR to add a revbump that adds support later.

Of note, JACK1 has a lot of keywords: ~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd. AIUI, that means I also need to get this tested by all of those arch teams and the keywords added, or else drop them from the revised JACK ebuild that actually adds support, right?

aballier commented on media-libs/libffado/metadata.xml:
```diff
@@ -0,0 +1,16 @@
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
```
please put proaudio@g.o there too

aballier commented:
looks very good to me; cant find any important comment :)

please add proaudio@g.o to metadata and I think it can be merged

aballier commented:
it would be nice if patches could have some links in their metadata to upstream bugreports or commits

marcan commented:
* Added proaudio to metadata.xml
* Added upstream commit info to `libffado-2.3.0-gcc6.patch`
* Replaced `libffado-2.3.0-modelname-strip.patch` with the equivalent upstream patch (mine was a quick fix for the same issue, downstream).

aballier commented:
merged, thx!
